docs/UserGuide.md

```diff
@@ -425,9 +421,10 @@ _Details coming soon ..._
 ## Known issues
 
 1. **When using multiple screens**, if you move the application to a secondary screen, and later switch to using only the primary screen, the GUI will open off-screen. The remedy is to delete the `preferences.json` file created by the application before running the application again.
-2. **view -stats Command**, Currently the total fee owing by students can precision more than 2 decimal points
-3. **When inputting long names above 220 characters**, the Id number of the students will be blocked even after the work around ontop[in add command](#adding-a-student-add)
-4. **When using the payment commands**, the display will round the payment amounts to the nearest $0.01. So if you enter a payment of $0.001, it will be displayed as $0.00. However, the application will accurately track the exact amounts entered, without rounding.
+2. **When inputting long names above 220 characters**, the Id number of the students will be blocked even after the work around ontop[in add command](#adding-a-student-add)
```

Comment:
standardise Id to ID, and change ontop[in add command] to like [on top](#adding-a-student-add)